src/test/store/receive-profile.test.js

```diff
@@ -66,13 +74,23 @@ describe('actions/receive-profile', function() {
         getSymbolTable: () => Promise.resolve(),
       };
       window.geckoProfilerPromise = Promise.resolve(geckoProfiler);
+
+      // This is a mock implementation because of the `mock` call above, but
+      // Flow doesn't know this.
+      (SymbolStoreDB: any).mockImplementation(() => ({
+        getSymbolTable: jest.fn().mockResolvedValue(exampleSymbolTable),
+      }));
+
+      window.TextDecoder = TextDecoder;
```

Comment:
Various issues here, coming from the fact that we don't have indexedDB in this test file, and so SymbolStoreDB was broken. The errors weren't transmitted back to the test, but because in `SymbolStore._getSymbolTable` we're storing stuff in the database but not waiting for it in the promise, the errors were "leaking" in other tests.

So I took the path of:
1. Throwing back unexpected errors to the top level calls, failing the right tests
2. Fixing the tests by mocking SymbolStoreDB itself, making it returning a proper value so that we don't try to store data, etc. We don't need this in this test.

This proved quite a lot of work :)